forge/routes/api/application.js

```diff
@@ -26,6 +28,8 @@ module.exports = async function (app) {
         }
     })
 
+    app.register(applicationDeviceGroup, { prefix: '/:applicationId/devicegroups' })
```

Comment:
```suggestion
    app.register(applicationDeviceGroup, { prefix: '/:applicationId/device-groups' })
```
Elsewhere we use hyphen-separated words for the api parts. I'm proposing this change here, but obviously will need updating throughout.